odata-url-conventions/4 Resource Path.md

```diff
@@ -998,4 +999,41 @@ Content-Type: text/plain
 
 $filter=[FirstName,LastName]%20in%20[["John","Doe"],["Jane","Smith"]]
 ```
+This POST request would result from submitting the HTML form
+```html
+<form method="post" action="http://host/service/People/$query"
+      enctype="text/plain">
+  <input name="$filter"
+    value='[FirstName,LastName]%20in%20[["John","Doe"],["Jane","Smith"]]'>
+</form>
+```
+:::
+
+For `Content-Type: application/x-www-form-urlencoded`, the request body MUST be
+the value of _output_ after running the
+[`application/x-www-form-urlencoded` serializer](https://url.spec.whatwg.org/#concept-urlencoded-serializer)
+in the [URL Living Standard](#_url), section 5.2, with _tuples_ being the list
+of name/value pairs for the individual query options.
```

Comment:
Can we also use this reference to replace the second half of [OData-URL, section 2.1]?

Reply:
No, the WhatWG URL Spec is in parts contradicting RFC3986, which is why we spell out the "safe" compromise

> Note: neither [RFC3986](https://docs.oasis-open.org/odata/odata/v4.02/csd01/part2-url-conventions/odata-v4.02-csd01-part2-url-conventions.html#rfc3986) nor this specification assign special meaning to “+” (octet 0x2B). Some implementations decode “+” (octet 0x2B) as space (octet 0x20), others take it literally.
> 
> Clients SHOULD percent-encode space (octet 0x20) as %20 and “+” (octet 0x2B) as %2B and avoid the ambiguous “+” (octet 0x2B) in URLs.

Reply:
We also added to protocol

> [12.3 Interoperable OData Clients](https://docs.oasis-open.org/odata/odata/v4.02/csd01/part1-protocol/odata-v4.02-csd01-part1-protocol.html#InteroperableODataClients)
> ...
> 10. MUST encode the plus character (octet `0x2B`) as `%2B` in URLs to avoid servers mis-interpreting the plus character as an encoded space